### PR TITLE
TJJ-030: Created client and client address model

### PR DESCRIPTION
**Pull request by dylanmckk**

### CODE REVIEW

File: src/main/java/org/example/models/Client.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        this.clientAddress.setAddressCity(addressCity);
+    }
+
+    public String getClientAddressProvince() {
+        return clientAddress.getAddressProvince();
+    }
+
+    public void setClientAddressProvince(final String addressProvince) {
+        this.clientAddress.setAddressProvince(addressProvince);
+    }
```
Comment (lukemKainos) [inline src/main/java/org/example/models/Client.java:74]: why have we added client to the function name

File: src/main/java/org/example/models/Client.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    public void setClientAddressLine2(final String addressLine2) {
+        this.clientAddress.setAddressLine2(addressLine2);
+    }
+
+    public String getClientAddressCity() {
+        return clientAddress.getAddressCity();
+    }
+
+    public void setClientAddressCity(final String addressCity) {
```
Comment (lukemKainos) [inline src/main/java/org/example/models/Client.java:64]: why is this here even though its been abstracted

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by dylanBkainos — APPROVED** [review]
Naming conventions all match 👍

**Review by lukemKainos — APPROVED** [review]
